uktrade/data-hub-frontend
# FEATURE/CLS2-1012 EYB lead header status message

## Description of change

This pr addresses [cls2-1012](https://uktrade.atlassian.net/browse/CLS2-1012)

The aim is to add a blue status message at the top of the page to show when a EYB lead has an investment project associated to it. The project should be linked as well.

## Test instructions

- Create a EYB lead
- Assign it a project
- When you visit the EYB lead details page, the banner should be visible and link to the project's detail page

If you add the project via the provided "Add investment project" button, it's expected for it to be hidden afterwards.

## Screenshots

### Without a project

<img width="1020" alt="Screenshot 2024-11-08 at 13 57 55" src="https://github.com/user-attachments/assets/2a761773-998c-4a1e-8d1b-1c120c35a6b1">


### With a project

<img width="1020" alt="Screenshot 2024-11-08 at 14 25 17" src="https://github.com/user-attachments/assets/79ed7f0f-d9f0-4d4f-aba6-e01409920552">


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
